server/core/region.go

```diff
@@ -165,18 +168,21 @@ func RegionFromHeartbeat(heartbeat *pdpb.RegionHeartbeatRequest, opts ...RegionC
 	return region
 }
 
+// Inherit inherits the buckets and region size from the parent region.
 // CorrectApproximateSize correct approximate size by the previous size if here exists an reported RegionInfo.
```

Comment:
So what is `CorrectApproximateSize`, should we remove it?

Reply:
the comment will be " correct approximate size and buckets by the previous size if here exists a reported RegionInfo."